ui/app/js/controllers/rule_new_controller.js

```diff
@@ -25,10 +25,20 @@ function($scope, $http, $modalInstance, CSRF, Releases, Rules, rules, rule, pr_c
   $scope.saveChanges = function () {
     $scope.saving = true;
     $scope.errors = {};
+    $scope.rate_error = {};
```

Comment:
This looks unused. Please remove it if that's the case.

Reply:
Removed. It was a relic from previous attempts at a solution which I neglected to remove. Good catch.